test/optimizer/cost_test.cpp

```diff
@@ -42,7 +42,7 @@ namespace test {
 
 using namespace optimizer;
 
-const int N_ROW = 100;
+// const int N_ROW = 100;
```

Comment:
If this isn't being used, remove it entirely.

Reply:
The entire test is commented out except for this definition which was giving me a compilation error. So I just commented this out as well.

Reply:
Is the test obsolete now? If so, why not delete it too. We have a lot of commented out code in this file.

Reply:
I think this was commented out in #902. @chenboy I believe said he would make changes to uncomment it?

Reply:
If this is all that's blocking the PR, I'd leave it. Getting the fix in is more important.